### PR TITLE
Githubtraining adding file and showing an example

### PR DESCRIPTION
**Pull request by nikhilbhambure**
This file is for testing and used to show/test basic use of git
@chandreshsharma This is just a testing. Please review :) !!!

### CODE REVIEW

File: nikhil-mumbai.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+"Hello world"
+## This is test
+
```
Comment (codeShar) [inline nikhil-mumbai.md]: Add in some great places to eat!

Comment (codeShar) [inline nikhil-mumbai.md]: Add places to visit!

Comment (nikhilbhambure) [inline nikhil-mumbai.md]: @codeShar Sure I will

Comment (nikhilbhambure) [inline nikhil-mumbai.md]: @codeShar added couple of places in Mumbai to visit along with the food

**Pull request merged**